_source/_posts/2025-09-03-cross-app-access.md

```diff
@@ -0,0 +1,385 @@
+   * Once created, sign in to your new org
+
+2. **AWS Credentials:** You'll need an **AWS Access Key ID** and **AWS Secret Access Key**  
+   * The IAM user or role associated with these credentials must have access to **Amazon Bedrock,** specifically the **Claude 3.7 Sonnet model,** enabled  
+   * If you don't know how to obtain the credentials, [follow this guide](https://github.com/oktadev/okta-cross-app-access-mcp/blob/main/guide/aws-bedrock.md)
+
+3. **Developer Tools:** These tools are essential for cloning, editing, building, and running your demo applications  
+   * **[Git](https://git-scm.com/downloads)** – to clone and manage the repository  
+   * **[VS Code](https://code.visualstudio.com/Download)** – for reading and modifying the sample source code  
+   * **[Dev Containers Extension (VS Code)](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)** – recommended, as it automatically configures dependencies and environments when you open the project  
+   * **[Docker](https://www.docker.com/products/docker-desktop/)** – required by the Dev Container to build and run the sample applications in isolated environments
+
+**Table of Contents**{: .hide }
+
+* Table of Contents
+{:toc}
+
+## Use Okta to secure AI applications with OAuth 2.0 and OpenID Connect (OIDC)
+
+Before we dive into the code, we need to register our apps with Okta. In this demo:
+
+* **Agent0**: the AI agent **requesting app** (makes the API call on behalf of the user)
+* **Todo0**: the **resource app** (owns the protected API)
+* **Managed connection**: the trust relationship between the two apps, created in Okta
+
+We'll create both apps in your Okta Integrator Free Plan account, grab their client credentials, and then connect them.
+
+### Enable Cross App Access in your Okta org
+
+> ⚠️ **Note:** Cross App Access is currently a **self-service Early Access (EA) feature**. It must be enabled through the Admin Console before the apps appear in the catalog. If you don't see the option right away, refresh and confirm you have the necessary admin permissions. Learn more in the [Okta documentation on managing EA and beta features](https://help.okta.com/oie/en-us/content/topics/security/manage-ea-and-beta-features.htm).
+
+1. Sign in to your Okta Integrator Free plan account 
+2. In the **Okta Admin Console**, select **Settings > Features**  
+3. Navigate to **Early access**  
+4. Find **Cross App Access** and select **Turn on** (enable the toggle)  
+5. Refresh the Admin Console
+
+{% img blog/cross-app-access/image9.jpg alt:"Enable Cross App Access feature in Okta Admin Console" width:"800" %}{: .center-image }
+
+### Create the resource app (Todo0)
+
+1. In the Okta Admin console, navigate to **Applications > Applications**  
+2. Select **Browse App Catalog**  
+3. Search for **Todo0 - Cross App Access (XAA) Sample Resource App**, and select it  
```

Comment:
Should it be Search for "Todo0 - ...."

see comment below

Reply:
This will also work but i felt let's add full name as of now so that it would be accurate